### DISCUSSION TITLE
How to prioritise issues?

### DISCUSSION THREAD

**Discussion by mxochicale** [UCL-ARC/python-tooling#242] [General]
Hi,

There might be a different ways to prioritise software issues but let's start with this reference: https://www.softwaretestinghelp.com/how-to-set-defect-priority-and-severity-with-defect-triage-process/ 

So, I have just added the following [label priorities](https://github.com/UCL-ARC/python-tooling/labels):
* `p1-critical` > This has to be fixed immediately within 24 hours.
* `p2-high`
* `p3-medium`
* `p4-low`

Do you have other suggestions or references? 
Do we want to have milestones associated with such labels?

Thanks, Miguel

**Comment by samcunliffe** [comment]
Yeah looks good and sane to me. I'm also a fan of the shades of red for the labels.
